selfhosted

```diff
@@ -3,9 +3,10 @@
 # Script for running the selfhosted tests on QPUs directly from GitHub
 # Tests need to be copied to /tmp/ because coverage does not work with NFS
 
-cp -r tests /tmp/
-cp pyproject.toml /tmp/
-cd /tmp/tests
+tmpdir=$(mktemp -d)
```

Comment:
```suggestion
tmpdir=$(mktemp -d selfhosted.qibo.XXXXXXX)
```

Just because it's possible, to better identify them (in case it will not be removed, because the workflow fails before for any reason).